misc/msvc/q3_ui.def

```diff
@@ -1,3 +0,0 @@
-EXPORTS
-	vmMain
-	dllEntry
```

Comment:
It will always be in git itself. Untill we figure out how to build with VS under Windows i think its safe to delete these.